include/xten/Dialect/XTenNN/IR/XTenNNOps.td

```diff
@@ -148,4 +148,20 @@ def XTenNN_DequantizeOp: XTenNN_Op<"dequantize", [
   let assemblyFormat = [{ `(`$input `:` type($input)`)` attr-dict `->` type($output) }];
 }
 
+def XTenNN_LoadExternalConstOp: XTenNN_Op<"load_external_const", [
```

Comment:
Could you make it so that it has `ConstantLike` trait?
```suggestion
def XTenNN_LoadExternalConstOp: XTenNN_Op<"load_external_const", [ ConstantLike
```